src/v2i-hub/TelematicBridgePlugin/src/TelematicBridgePlugin.h

```diff
@@ -3,21 +3,30 @@
 
 #include "PluginClient.h"
 #include "TelematicBridgeMsgWorker.h"
+#include "TelematicUnit.h"
 
 using namespace tmx::utils;
```

Comment:
Using namespace is bad practice especially in header files.

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/567/commits/71ac033b94eec7ea112cb5ead2d680a29b769772